test/remote/gateways/remote_redsys_sha256_test.rb

```diff
@@ -212,6 +212,22 @@ def test_successful_3ds2_purchase_with_mit_exemption
     assert_match 'MIT', response.params['ds_excep_sca']
   end
 
+  def test_successful_ntid_generation_with_verify
```

Comment:
perhaps you can add a comment here to explain that the gateway's sandbox does not actually generate the ntid that we expect to see in production.